src/libraries/System.DirectoryServices.Protocols/tests/BerConverterTests.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace System.DirectoryServices.Protocols.Tests
 {
+    [ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsOpenSUSE))]
```

Comment:
Wouldn't this make the tests ONLY run on OpenSUSE?

Reply:
That explains why my build is green all of a sudden :). dumb error, will fix.